test/json_mapper_helper_test.dart

```diff
@@ -3,42 +3,124 @@ import 'package:workmanager/src/options.dart';
 import 'package:workmanager/src/workmanager.dart';
```

Comment:
Woow.
Did you rearrange this file?

Reply:
I added a `group("toInitializeMethodArgument")` and moved existing tests to a new `group("toRegisterMethodArgument")`

Reply:
the diff fucks up bad time